src/client/src/rt-platforms/openFin/adapter/window.ts

```diff
@@ -74,32 +74,42 @@ export function createPlatformWindow(getWindow: () => Promise<fin.OpenFinWindow>
   }
 }
 
-export const openDesktopWindow = async (
-  config: DesktopWindowProps,
-  onClose?: () => void,
-  position?: {},
-): Promise<PlatformWindow> => {
-  const { url, width: defaultWidth, height: defaultHeight, maxHeight, maxWidth } = config
-  const minWidth = config.minWidth ? config.minWidth : 100
-  const minHeight = config.minHeight ? config.minHeight : 100
+type OpenfinWindowPosition = Pick<fin.WindowOption, 'defaultLeft' | 'defaultTop'>;
 
-  const childWindows = await getChildWindows()
-  const hasChildWindows = childWindows && childWindows.length > 0
-  const shouldBeDefaultCentered = !hasChildWindows && (!config.x && !config.y)
-
-  let updatedPosition = {
-    defaultLeft: config.x ? config.x : undefined,
-    defaultTop: config.y ? config.y : undefined,
+async function getOpenfinWindowPosition(config: DesktopWindowProps, childWindows?: fin.OpenFinWindow[]): Promise<OpenfinWindowPosition> {
```

Comment:
moving calculating position to a new function. Here the change is to NOT do cascading windows when there are explicit coordinates from the function call